src/services/action/repositories/action.ts

```diff
@@ -99,7 +99,13 @@ export class ActionRepository {
    */
   async getForItem(
     itemPath: UUID,
-    filters?: { sampleSize?: number; view?: string; memberId?: UUID },
+    filters?: {
+      sampleSize?: number;
+      view?: string;
+      memberId?: UUID;
+      startDate?: string;
+      endDate?: string;
```

Comment:
I think it needs default values. That would be nice to have one month to now as defaults.